app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -48,7 +42,7 @@ private fun Button(
         onClick = onClick,
         modifier = Modifier
             .fillMaxWidth()
-            .height(36.dp),
+            .padding(horizontal = 16.dp, vertical = 12.dp),
```

Comment:
So Composables are a little tricky with padding. Sometimes just doing `Modifier.padding(..)` is enough. Other times, when using Composables in the Material library, we have to use the `contentPadding` API that those components expose. In this case, `androidx.compose.material.Button` has a parameter called `contentPadding` that we want to use here. In essence, the padding addition should look like: 

``` 
androidx.compose.material.Button(
    ...
    modifier = Modifier.fillMaxWidth(),
    ...
    contentPadding = PaddingValues(horizontal = 16.dp, vertical = 12.dp)
) {
```

If we didn't use `contentPadding`, the padding applied via the Button's modifier would instead be _outside_ the Button's drawing area. We want this padding _inside_ the Button.